vita3k/app/src/app_init.cpp

```diff
@@ -319,10 +319,18 @@ bool init(EmuEnvState &state, Config &cfg, const Root &root_paths) {
         window_type |= SDL_WINDOW_ALLOW_HIGHDPI;
         state.dpi_scale = ddpi / 96;
     }
+#elif defined(__APPLE__)
+    window_type |= SDL_WINDOW_ALLOW_HIGHDPI;
+    state.dpi_scale = 2; // It is fixed value on macOS.
 #endif
     state.res_width_dpi_scale = static_cast<uint32_t>(DEFAULT_RES_WIDTH * state.dpi_scale);
     state.res_height_dpi_scale = static_cast<uint32_t>(DEFAULT_RES_HEIGHT * state.dpi_scale);
+
+#ifndef __APPLE__
     state.window = WindowPtr(SDL_CreateWindow(window_title, SDL_WINDOWPOS_CENTERED, SDL_WINDOWPOS_CENTERED, state.res_width_dpi_scale, state.res_height_dpi_scale, window_type | SDL_WINDOW_RESIZABLE), SDL_DestroyWindow);
+#else
+    state.window = WindowPtr(SDL_CreateWindow(window_title, SDL_WINDOWPOS_CENTERED, SDL_WINDOWPOS_CENTERED, DEFAULT_RES_WIDTH, DEFAULT_RES_HEIGHT, window_type | SDL_WINDOW_RESIZABLE), SDL_DestroyWindow);
+#endif
```

Comment:
I think calling SDL_CreateWindow with  DEFAULT_RES_WIDTH and  DEFAULT_RES_HEIGHT should be the correct behavior on all platforms